modules/ROOT/pages/architecture.adoc

```diff
@@ -42,6 +42,36 @@ The minimum requirements are 3 controller and 3 worker nodes. 3 controllers enab
 
 Mule applications run on workers. Multiple replicas of applications can run across workers.
 
+== Scaling Considerations
+
+* Minimum required number of controller nodes. 
```

Comment:
Let's add max number of controller nodes as well, which is 5.
Also useful is a note to ensure customers don't provision an even number of controller nodes (2 or 4) -- as that doesn't improve fault-tolerance.

Reply:
Added - thank you!